### PR TITLE
Added Python Solution for Concert Tickets

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/silver/cses-1091.mdx [hunk 1]
```diff
@@ -162,4 +162,38 @@ public class ConcertTickets {
 }
 ```
 </JavaSection>
+
+<PySection>
+
+```py
+# Author: NuttyJamie
+
```
Comment (SansPapyrus683) [inline solutions/silver/cses-1091.mdx:170]: there's an author field at the top of the file

File: solutions/silver/cses-1091.mdx [hunk 2]
```diff
@@ -162,4 +162,38 @@ public class ConcertTickets {
...
+import sys
+from bisect import *
+
+sys.setrecursionlimit(200000)
+
+
+def get_endpoint(idx):
```
Comment (SansPapyrus683) [inline solutions/silver/cses-1091.mdx:177]: type annotations

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Comment by SansPapyrus683** [comment]
massive
can you send screenshot of ac

**Comment by freakin23** [comment]
> massive can you send screenshot of ac

Screenshot from 2024-05-16 22-17-52

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]

**Comment by ryanchou-dev** [comment]
awesome, thanks!

**Pull request merged**